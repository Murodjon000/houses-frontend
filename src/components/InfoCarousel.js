import React from 'react';
import { Carousel } from 'react-bootstrap';
import man from '../assets/man.jpg';
import man2 from '../assets/man-2.jpg';
import women from '../assets/women.jpg';

const InfoCarousel = () => (
  <div className=" infoCarousel__wrapper pt-5">
    <h2 className="text-sm-gray text-center my-2 text-uppercase fw-bold">
      People are already enjoying the service
    </h2>
    <h1 className="text-lg-black text-center mb-3">
      Look what these have to say
    </h1>
    <Carousel className="mb-3">
      <Carousel.Item>
        <div className="shadow rounded infoCarousel__wrapper-desc-info p-4 mx-auto ">
          <img
            src={man}
            className="d-block mx-auto card-rounded-image"
            alt="..."
          />
          <h3 className="text-center fw-bold">John Hudson</h3>
          <p className="text-sm-gray text-center">Designer</p>
          <p className="text-sm-gray text-center">
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus,
            doloremque! Harum repellendus vero voluptas exercitationem?
          </p>
        </div>
      </Carousel.Item>
      <Carousel.Item>
        <div className="shadow rounded  infoCarousel__wrapper-desc-info p-4 mx-auto">
          <img
            src={man2}
            className="d-block mx-auto card-rounded-image"
            alt="..."
          />
          <h3 className="text-center fw-bold">Dean Hudson</h3>
          <p className="text-sm-gray text-center">Lawyer</p>
          <p className="text-sm-gray text-center">
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus,
            doloremque! Harum repellendus vero voluptas exercitationem?
          </p>
        </div>
      </Carousel.Item>
      <Carousel.Item>
        <div className="shadow rounded infoCarousel__wrapper-desc-info p-4 mx-auto">
          <img
            src={women}
            className="d-block mx-auto card-rounded-image my-2"
            alt="..."
          />
          <h3 className="text-center fw-bold">Emma Watson</h3>
          <p className="text-sm-gray text-center my-2">Designer</p>
          <p className="text-sm-gray text-center">
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus,
            doloremque! Harum repellendus vero voluptas exercitationem?
          </p>
        </div>
      </Carousel.Item>
    </Carousel>
  </div>
);

export default InfoCarousel;
