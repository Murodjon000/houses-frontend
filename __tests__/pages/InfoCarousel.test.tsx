import React from 'react';
import renderer from 'react-test-renderer';
import InfoCarousel from '../../components/mainPage/InfoCarousel';

test('should correctly render Details page', () => {
  const component = renderer.create(<InfoCarousel />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
