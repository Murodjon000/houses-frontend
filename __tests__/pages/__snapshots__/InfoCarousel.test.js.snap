// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Details page 1`] = `
<div
  className=" infoCarousel__wrapper pt-5"
>
  <h2
    className="text-sm-gray text-center my-2 text-uppercase fw-bold"
  >
    People are already enjoying the service
  </h2>
  <h1
    className="text-lg-black text-center mb-3"
  >
    Look what these have to say
  </h1>
  <div
    className="mb-3 carousel slide"
    onKeyDown={[Function]}
    onMouseOut={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
  >
    <ol
      className="carousel-indicators"
    >
      <li
        className="active"
        onClick={[Function]}
      />
      <li
        onClick={[Function]}
      />
      <li
        onClick={[Function]}
      />
    </ol>
    <div
      className="carousel-inner"
    >
      <div
        className="active carousel-item"
      >
        <div
          className="shadow rounded infoCarousel__wrapper-desc-info p-4 mx-auto "
        >
          <img
            alt="..."
            className="d-block mx-auto card-rounded-image"
            src="man.jpg"
          />
          <h3
            className="text-center fw-bold"
          >
            John Hudson
          </h3>
          <p
            className="text-sm-gray text-center"
          >
            Designer
          </p>
          <p
            className="text-sm-gray text-center"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus, doloremque! Harum repellendus vero voluptas exercitationem?
          </p>
        </div>
      </div>
      <div
        className="carousel-item"
      >
        <div
          className="shadow rounded  infoCarousel__wrapper-desc-info p-4 mx-auto"
        >
          <img
            alt="..."
            className="d-block mx-auto card-rounded-image"
            src="man-2.jpg"
          />
          <h3
            className="text-center fw-bold"
          >
            Dean Hudson
          </h3>
          <p
            className="text-sm-gray text-center"
          >
            Lawyer
          </p>
          <p
            className="text-sm-gray text-center"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus, doloremque! Harum repellendus vero voluptas exercitationem?
          </p>
        </div>
      </div>
      <div
        className="carousel-item"
      >
        <div
          className="shadow rounded infoCarousel__wrapper-desc-info p-4 mx-auto"
        >
          <img
            alt="..."
            className="d-block mx-auto card-rounded-image my-2"
            src="women.jpg"
          />
          <h3
            className="text-center fw-bold"
          >
            Emma Watson
          </h3>
          <p
            className="text-sm-gray text-center my-2"
          >
            Designer
          </p>
          <p
            className="text-sm-gray text-center"
          >
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Minus, doloremque! Harum repellendus vero voluptas exercitationem?
          </p>
        </div>
      </div>
    </div>
    <a
      className="carousel-control-prev"
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-prev-icon"
      />
      <span
        className="sr-only"
      >
        Previous
      </span>
    </a>
    <a
      className="carousel-control-next"
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-next-icon"
      />
      <span
        className="sr-only"
      >
        Next
      </span>
    </a>
  </div>
</div>
`;
